docs/02.API-REFERENCE.md

```diff
@@ -1824,6 +1824,11 @@ enough for the whole string.
 *Note*: Does not put '\0' to the end of string, the return value identifies
 the number of valid bytes in the output buffer.
 
+*Note*: If jerry value is larger than target buffer, then it will fail to copy.
+Use [jerry_substring_to_char_buffer](#jerry_substring_to_char_buffer) instead if
```

Comment:
`To copy a substring the [jerry_substring_to_char_buffer](#jerry_substring_to_char_buffer) API function is recommended instead.`

Reply:
Please also fix the other comments.